components/eamxx/docs/field_aliasing.md

```diff
@@ -0,0 +1,128 @@
+# EAMxx Field Aliasing Feature
+
+This document demonstrates the new field aliasing feature for EAMxx I/O operations.
```

Comment:
Avoid words like "new feature", as that is only relevant now, but won't make sense in a few months/years.